build.sh

```diff
@@ -267,6 +266,37 @@ else
     export LIBRARY_PATH=${JAVA_HOME}/jre/lib/amd64/server/
 fi
 
+build_submodules()
```

Comment:
better to extract them into `common.sh`, it's duplicated in `run-be-ut.sh` too.

Reply:
> better to extract them into `common.sh`, it's duplicated in `run-be-ut.sh` too.

OK